movement/plots/occupancy.py

```diff
@@ -0,0 +1,185 @@
+
+    - If there are multiple keypoints selected, the occupancy of the centroid
+        of these keypoints is computed.
+    - If there are multiple individuals selected, the occupancies of their
+        centroids are aggregated.
+
+    Points whose corresponding spatial coordinates have NaN values
+    are ignored.
+
+    Histogram information is returned as the third output value (see Notes).
+
+    Parameters
+    ----------
+    da : xarray.DataArray
+        Spatial data to create histogram for. NaN values are dropped.
+    individuals : Hashable, optional
+        The name of the individual(s) to be aggregated and plotted. By default,
+        all individuals are aggregated.
+    keypoints : Hashable | list[Hashable], optional
+        Name of a keypoint or list of such names. The centroid of all provided
+        keypoints is computed, then plotted in the histogram.
+    ax : matplotlib.axes.Axes, optional
+        Axes object on which to draw the histogram. If not provided, a new
+        figure and axes are created and returned.
+    kwargs : Any
+        Keyword arguments passed to ``matplotlib.pyplot.hist2d``
+
+    Returns
+    -------
+    matplotlib.pyplot.Figure
+        Plot handle containing the rendered 2D histogram. If ``ax`` is
+        supplied, this will be the figure that ``ax`` belongs to.
+    matplotlib.axes.Axes
+        Axes on which the histogram was drawn. If ``ax`` was supplied,
+        the input will be directly modified and returned in this value.
+    dict[str, numpy.ndarray]
+        Information about the created histogram (see Notes).
+
+    Notes
+    -----
+    In instances where the counts or information about the histogram bins is
+    desired, the ``return_hist_info`` argument should be provided as ``True``.
+    This will force the function to return a second output value, which is a
+    dictionary containing the bin edges and bin counts that were used to create
+    the histogram.
+
+    For data with ``Nx`` bins in the 1st spatial dimension, and ``Ny`` bins in
+    the 2nd spatial dimension, the dictionary output has key-value pairs;
+    - ``xedges``, an ``(Nx+1,)`` ``numpy`` array specifying the bin edges in
+    the 1st spatial dimension.
+    - ``yedges``, an ``(Ny+1,)`` ``numpy`` array specifying the bin edges in
+    the 2nd spatial dimension.
+    - ``counts``, an ``(Nx, Ny)`` ``numpy`` array with the count for each bin.
+
+    ``counts[x, y]`` is the number of datapoints in the
+    ``(xedges[x], xedges[x+1]), (yedges[y], yedges[y+1])`` bin. These values
+    are those returned from ``matplotlib.pyplot.Axes.hist2d``.
+
+    Examples
+    --------
+    Simple use-case is to plot a histogram of the centroid of all
+    keypoints, aggregated over all individuals.
+
+    >>> from movement import sample_data
+    >>> from movement.plots import plot_occupancy
+    >>> positions = sample_data.fetch_dataset(
+    ...     "SLEAP_three-mice_Aeon_proofread.analysis.h5"
```

Comment:
I think this particualr dataset is the most confusing one for this example, because the animals contain a single keypont named "centroid". While reading the example, I had a hard time differentiating between "the centroid of given ketypints" and "the keypoints named centroid".

To save ourselves this trouble, I suggest using the "DLC_two-mice.predictions.csv" dataset instead. That once contains two individuals, with multiple keypoints each.

```suggestion
    ...     "DLC_two-mice.predictions.csv"
```